### PR TITLE
RotatingBody, a new class between MassiveBody and OblateBody in the body hierarchy

### PR DESCRIPTION
**Pull request by pleroy**
Complete with serialization and pre/post-Brouwer compatibility.

### CODE REVIEW

File: physics/rotating_body.hpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+namespace physics {
+
+template<typename Frame>
+class RotatingBody : public MassiveBody {
+  static_assert(Frame::is_inertial, "Frame must be inertial");
+
+ public:
+  class Parameters {
+   public:
+    Parameters(Angle const& reference_angle,
```
Comment (eggrobin) [inline physics/rotating_body.hpp]: Some documentation (describing the geometry or at least referring to the IAU WGCC report) would be welcome.

File: physics/rotating_body.hpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  RotatingBody(MassiveBody::Parameters const& massive_body_parameters,
+               Parameters const& parameters);
+  ~RotatingBody() = default;
+
+  // Returns the angular velocity passed at construction.
+  AngularVelocity<Frame> const& angular_velocity() const;
+
+  // Returnst the position at time |t|.
+  Angle AngleAt(Instant const& t) const;
+
```
Comment (eggrobin) [inline physics/rotating_body.hpp]: Add an accessor for the rotation exp(ω(t - t0)).

File: physics/rotating_body.hpp (hunk 1 above)
Comment (pleroy) [inline physics/rotating_body.hpp]: Done.

File: physics/rotating_body.hpp (hunk 2 above)
Comment (pleroy) [inline physics/rotating_body.hpp]: Done.

**Pull request merged**